nav2_bringup/params/nav2_params.yaml

```diff
@@ -42,7 +42,7 @@ bt_navigator:
   ros__parameters:
     global_frame: map
     robot_base_frame: base_link
-    odom_topic: /odom
```

Comment:
Comment: Regardless, I agree with this. These were added due to back-in-the-day issues with ROS 2 topics needing the full path which has long _long_ not been true